gprofiler/main.py

```diff
@@ -289,26 +306,40 @@ def _snapshot(self):
         except Exception:
             logger.error("Running perf failed; consider running gProfiler with '--perf-mode none' to avoid using perf")
             raise
-
+        metadata = (
+            get_current_metadata(self._static_metadata)
```

Comment:
Why sending the static metadata every time?

Reply:
Talked about this IRL (we can add an ID later to send the metadata once on boot for a new endpoint and get and ID which the agent will send on each profile upload)

Reply:
k. will need to maintain compatibility later.